runner/runner_test.go

```diff
@@ -0,0 +1,72 @@
+package runner_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/scootdev/scoot/runner"
+)
+
+func TestCommandStringSimple(t *testing.T) {
+	c := runner.Command{
+		Argv:       []string{"cat", "cats.in", "|", "cat.bin", ">", "cats.out"},
```

Comment:
If tests double as documentation, I don't think we want to pipe/redirect here. AFAIK that's not supported in golang unless it's a single arg passed to a bash binary.